pkgs/development/libraries/stellarsolver/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, mkDerivation, fetchFromGitHub, cmake,
+  qtbase, cfitsio, gsl, wcslib }:
+
+mkDerivation rec {
+  pname = "stellarsolver";
+  version = "1.5";
+
+  src = fetchFromGitHub {
+    owner = "rlancaste";
+    repo = "stellarsolver";
+    rev = "${version}";
+    sha256 = "12j20j9qbkkx55ix4nm1iw7wd36hamkpidbwhcnmj4yk5fqlyq4y";
+  };
+
+  nativeBuildInputs = [
+    cmake
+  ];
+
+  buildInputs = [
+    qtbase
+    cfitsio
+    gsl
+    wcslib
+  ];
+
+  cmakeFlags = [
+    "-DBUILD_TESTER=off"
```

Comment:
Why did you disable this?

Reply:
Its off by default in CMakeLists.txt upstream, my latest commit makes it an option (withTester) that can be passed to the derivation, defaulting to off. I tested it both on and off and it checks out.